### ISSUE TITLE
Should all steps be separated with an empty new line?

### ISSUE THREAD

**Issue by gaborbernat** [FFY00/python-build#51]
My vote is for not. I'd only separate in logical blocks, mostly check sets.

**Comment by FFY00** [comment]
I vote yes, IMO it makes it easier to distinguish the steps. It more sense particularly when we have long steps (complex configuration).

**Comment by gaborbernat** [comment]
Most configuration though is simple and all the interrupt of empty lines makes it hard to follow steps that only make sense together.
